package.json

```diff
@@ -57,8 +57,8 @@
     "karma-webpack": "2.0.3",
     "pica": "2.0.8",
     "react": "15.4.2",
-    "react-addons-css-transition-group": "15.4.2",
     "react-addons-test-utils": "15.4.2",
+    "react-transition-group": "1.2.0",
```

Comment:
Sure, I was a bit confused with the double reference at first. I have made it a peerDependency, seems more relevant to manage it in own projects.